src/signals/incident-management/containers/IncidentOverviewPage/components/SubNav/index.js

```diff
@@ -15,6 +15,8 @@ export const MapHeading = styled(Heading).attrs({
 })`
   margin: 0;
   font-size: 16px;
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  font-weight: 400;
```

Comment:
These changes are not necessary; font is already `Demi`

Reply:
Unfortunately it is, it inherits `font-weight: 700`  from `Heading`

Reply:
I can confirm this (the inheritance).